mobile/ebitenmobileview/input_ios.go

```diff
@@ -58,3 +59,28 @@ func UpdateTouchesOnIOS(phase int, ptr int64, x, y int) {
 		panic(fmt.Sprintf("ebitenmobileview: invalid phase: %d", phase))
 	}
 }
+
+func UpdatePressesOnIOS(phase int, keyCode int, keyString string) {
+	switch phase {
+	case C.UITouchPhaseBegan, C.UITouchPhaseMoved, C.UITouchPhaseStationary:
+		if key, ok := iosKeyToUIKey[keyCode]; ok {
+			keys[key] = struct{}{}
+		}
+		if phase == C.UITouchPhaseBegan {
+			for _, r := range keyString {
+				if !unicode.IsPrint(r) {
+					continue
+				}
+				runes = append(runes, r)
+			}
+		}
+		updateInput()
+	case C.UITouchPhaseEnded, C.UITouchPhaseCancelled:
+		if key, ok := iosKeyToUIKey[keyCode]; ok {
+			delete(keys, key)
+		}
+		updateInput()
```

Comment:
```go
runes = nil
updateInput()
```

Reply:
Done that, but had to add this to EVERY callk site then.